sandbox/samples/python/basic/4_point_lights_test_scene.py

```diff
@@ -0,0 +1,240 @@
+
```

Comment:
I would suggest moving these Python programs to the test suite, next to the appleseed scenes they generate.

Also I believe the file `data/Plane001.binarymesh` is redundant.